src/components/dialog/dialog.scss

```diff
@@ -16,7 +17,7 @@
  * @prop --dialog-max-width: Max width of the dialog.
  * @prop --dialog-max-height: Max height of the dialog.
  * @prop --dialog-border-radius: Border radius of the dialog corners
- * @prop --dialog-padding-top-bottom: Padding on top and bottom of dialog content. Defaults to `1.5rem`.
+ * @prop --dialog-padding-top-bottom: Padding on top and bottom of dialog content. Defaults to `1.5rem`. Note that if you use this variable and set it to numbers smaller than 1rem, you will loose the edge fade effects when the content is scrollable and has overflowed out of the dialog's content area. This should be however fine! Because you must then be handling the overflow internally in the component that is sent to the dialog's content.
```

Comment:
Maybe I should also explain how these affect the fade-out effect of the edges, generally.

Reply:
I think this is a good explanation. Maybe we could add the preferred padding ranges.

Reply:
Maybe. The more padding, the larger the fade-out effect will be.